packages/table/src/table2.tsx

```diff
@@ -430,6 +430,29 @@ export class Table2 extends AbstractComponent2<Table2Props, TableState, TableSna
         this.quadrantStackInstance.scrollToPosition(correctedScrollLeft, correctedScrollTop);
     }
 
+    /**
+     * Scrolls the table by the provided x and y offset.
+     *
+     * - xOffset: The horizontal scroll modification
+     * - yOffset: The vertical scroll modification
+     *
+     */
+    public scroll(xOffset: number, yOffset: number) {
```

Comment:
I think this should be `{ left: number, top: number }` to match the `Offset` type:

https://github.com/palantir/blueprint/blob/3c58026dbfdade2f573a81eb2aa4d85af573ed01/packages/popover2/src/contextMenu2Shared.ts#L21-L24

I will make a change later to reconcile them as the same type, but for now they should be assignable to each other for API consistency across Blueprint.

```suggestion
    public scroll(relativeOffset: { left: number; top: number }) {
```